PowerEditor/src/NppDarkMode.cpp

```diff
@@ -1487,7 +1485,7 @@ namespace NppDarkMode
 
 			case WM_NCDESTROY:
 			{
-				::RemoveWindowSubclass(hWnd, ButtonSubclass, g_buttonSubclassID);
+				::RemoveWindowSubclass(hWnd, ButtonSubclass, static_cast<UINT_PTR>(SubclassID::first));
```

Comment:
remove `UNREFERENCED_PARAMETER(uIdSubclass);` above and use `uIdSubclass` instead